Makefile

```diff
@@ -0,0 +1,37 @@
+ENVIRONMENT ?= betacloud
+
+VERSION_CEPH ?= pacific
+VERSION_MANAGER ?= latest
+VERSION_OPENSTACK ?= yoga
```

Comment:
I don't like having yet another file where all these defaults are getting defined and need to be changed every cycle. Can we make this less redundant somehow? (Though I admit that I have no good idea for that immediately, too.)

Reply:
I'll open a follow-up issue for it.